src/containers/App.tsx

```diff
@@ -218,6 +219,16 @@ const Content = (): JSX.Element => {
         }
     }, [location.pathname, isOnTavle, settings])
 
+    useEffect(() => {
+        if (isOnTavle && openTavle) {
+            if (settings[0]?.pageRefreshedAt) {
+                if (openTavle < settings[0]?.pageRefreshedAt) {
+                    window.location.reload()
+                }
+            }
+        }
```

Comment:
The nested `if` statements can be un-nested by using `&&` or by using two seperate `if` statements. `openTavle` cannot be undefined since it it initialized in the call to `useState`.

```suggestion
        const pageRefreshedAt = settings[0]?.pageRefreshedAt
        if (!pageRefreshedAt || !isOnTavle) return

        if (openTavle < pageRefreshedAt) window.location.reload()
```

Reply:
Fixed